### ISSUE TITLE
Request For Grade

### ISSUE THREAD

**Issue by hend0298** [hend0298/animal-rescue-website#3]
@charlie-quinn

During the process of coding this website, I encountered some struggles. One of the main challenges was the website navigation. Originally I wanted to use a wavy SVG as the navigation background, but it was giving me too much trouble. The main issue was being able to change the size of the SVG when the browser size adjusted so that the wave didn't look smushed on smaller screens and super stretched out on bigger screens. Also, I couldn't get the navigation background to overlap over the top of the banner image. With not a lot of time left to finish the website, I decided to change the navigation background to a simple blue rectangle instead of the original wavy SVG since it was giving me too many issues. Although I would've liked to do the wavy SVG as the nav background, I think the rectangle is simple and keeps the design clean. Another challenge I faced with this website was with the dog's profiles on the Rescues page. Since I added a border under each dog's profile, once the browser got larger, the borders got all unaligned. After asking for help, I added unit-content-distribute on each li and content-shrink and content-fill on the corresponding elements, which aligned all the dog profiles. The last main struggle I faced was the quote and button on top of the banner image on the Home page. On smaller screens, it was almost impossible to make it fit on top of the image without blocking the dog in the photo. So, my solution was to make the quote go underneath the banner image on smaller screens then have it move on top of it once the screen size gets big enough. 

I believe I deserve an A as a grade for this project because I applied everything we've learned this year correctly and effectively. The first thing we learned this term that I applied was SVGs. I used a dog bone SVG as an icon to help divide headings and paragraphs. I also used the same icon as a favicon for my website which is another thing we learned this term. Another new technique I applied was compressing images. I adjusted all the images sizes in Photoshop to make sure they were all as small as can be without losing quality. I also successfully applied the required elements that we learned last year such as a form, grids and accessibility. 

In conclusion, I deserve an A because I effectively applied what we've learned in web dev and as a result, my website is fast, responsive and accessible to all. Although I faced some struggles throughout the process, I always found a solution to make sure the website would work and look good in the end.

**Comment by charlie-quinn** [comment]
- [ ] fix typo in navigation linked to the `adopt.html` page

✅ `This portion of your final assignment has been graded. A marking rubric has been sent to your college email address.`

⚠️`Reminder: please close this issue once it has been resolved.`
